src/RestServer/Newtonsoft/Json/BigDecimalJsonConverter.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// BigDecimalJsonConverter.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Numerics;
+
+namespace Neo.Plugins.RestServer.Newtonsoft.Json
```

Comment:
Cal we move all of these converters to `Neo.Json` project? 

- More organized.
- Reusable.
- Two PR, easy to review.

Reply:
They use `NewtonSoft.Json` library.

Reply:
Orderd a new machine, will arrive two days later. My current machine is too weak.

BTW, how hard is it for you to build a Restful Mock testing system?

Reply:
Well you have to mock the asp.net core. Which can be a pain. After that it's pretty easy.

Reply:
Let's move it to `Neo.Json`, easier to approve by chunks

Reply:
> Let's move it to `Neo.Json`, easier to approve by chunks

Won't be able to do because of **_circular dependencies_**.